barretenberg/cpp/src/barretenberg/sumcheck/instance/prover_instance.cpp

```diff
@@ -195,7 +195,7 @@ void ProverInstance_<Flavor>::construct_databus_polynomials(Circuit& circuit)
     // Note: We do not utilize a zero row for databus columns
     for (size_t idx = 0; idx < circuit.public_calldata.size(); ++idx) {
         public_calldata[idx] = circuit.get_variable(circuit.public_calldata[idx]);
-        calldata_read_counts[idx] = circuit.get_variable(circuit.calldata_read_counts[idx]);
+        calldata_read_counts[idx] = circuit.calldata_read_counts[idx];
```

Comment:
This was an unrelated bug that was illuminated through other debugging on this branch.